### ISSUE TITLE
Add new parameter to convert function

### ISSUE THREAD

**Issue by FlorinProfeanuVEGO** [kartik1998/pdf-images#10]
I noticed that png generated by convert have transparent background, when converting PDFs with white background. This is not happening when using jpg, like this: 
convert -quiet -density 200 Test.pdf -quality 200 Test.jpg

I would like you to add one more parameter with the file extension.

public static convert(pdfPath: string, outputImgDir: string, outputImgName: string, outputImgExtension: string): any {
.........
outputImgPath + '/' + outputImgName + outputImgExtension,

I attached a test file and the command lines to convert them:

convert -quiet -density 200 Test.pdf -quality 200 Test.jpg
convert -quiet -density 200 Test.pdf -quality 200 Test.png

[Test.pdf](https://github.com/kartik1998/pdf-images/files/10888006/Test.pdf)

![Test](https://user-images.githubusercontent.com/95374759/222893195-3a59aad9-a54b-4fdb-913f-bbbc7add560f.png)

![Test](https://user-images.githubusercontent.com/95374759/222893216-d4e87cb2-7f04-4153-ac97-567f54514b0a.jpg)

**Comment by kartik1998** [comment]
Hi, Sorry about the late response i just noticed this issue. I'll push the patch in a bit. 
Thanks!

**Comment by kartik1998** [comment]
v1.2 will be released later this week.

**Comment by FlorinProfeanuVEGO** [comment]
I appreciate. Thank you.

**Comment by FlorinProfeanuVEGO** [comment]
Can you estimate when you can publish the latest version?

**Comment by kartik1998** [comment]
Hi @FlorinProfeanuVEGO sorry about the delay. v1.2.0 has been published now.

**Comment by FlorinProfeanuVEGO** [comment]
Hello,

Unfortunately, the jpg extension does not solve the transparency issue. I mean it removes transparency but for one example I am working with, it turns the white background of the PDF into black in JPG.

I found this link https://www.imagemagick.org/discourse-server/viewtopic.php?t=31014 that gives a solution for PDFs. Indeed, the below command works better:
convert -density 300x300 -units pixelsperinch Test.pdf -background white -alpha background -alpha off -antialias -compress none +adjoin Test.png

During my tests I found that this one is OK as well:
convert -density 300 -quality 100 Test.pdf -alpha background -alpha off Test.png

What I would like you to do, if you agree, is to add a generic parameter where we can add things like -alpha background -alpha off.
Obviously is up to the user to add correct options here.

Florin
